yt/utilities/lib/tests/test_alt_ray_tracers.py

```diff
@@ -5,12 +5,13 @@
 from yt.testing import amrspace
 from yt.utilities.lib.alt_ray_tracers import _cyl2cart, cylindrical_ray_trace
 
-left_grid = right_grid = amr_levels = center_grid = data = None
+left_grid = right_grid = amr_levels = center_grid = data = old_settings = None
```

Comment:
I'm not a fan of these all living on the same line. No harm done in keeping the existing one (though I'd also approve expanding it) but I'd rather have the new binding on a separate line.